python/semantic_kernel/connectors/memory/azure_cosmosdb_no_sql/azure_cosmos_db_no_sql_base.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+from azure.cosmos.aio import ContainerProxy, CosmosClient, DatabaseProxy
+from azure.cosmos.exceptions import CosmosResourceNotFoundError
+from azure.identity import DefaultAzureCredential
+
+from semantic_kernel.connectors.memory.azure_cosmosdb_no_sql.azure_cosmos_db_no_sql_settings import (
+    AzureCosmosDBNoSQLSettings,
+)
+from semantic_kernel.exceptions.memory_connector_exceptions import MemoryConnectorResourceNotFound
+from semantic_kernel.kernel_pydantic import KernelBaseModel
+from semantic_kernel.utils.experimental_decorator import experimental_class
+
+
+@experimental_class
+class AzureCosmosDBNoSQLBase(KernelBaseModel):
```

Comment:
how much of this is NoSQL specific, or could this be reused for a Cosmos MongoDB connector as well?

Reply:
I am not sure now. We can do some analysis when we get to Cosmos Mongo DB.